src/server/IoDispatchers.cpp

```diff
@@ -146,6 +146,20 @@ static bool _shouldAttemptHandoff(const Globals& globals,
                                   const CONSOLE_INFORMATION& gci,
                                   CONSOLE_API_CONNECTINFO& cac)
 {
+#ifdef __INSIDE_WINDOWS
+    const bool insideWindows = true;
+#else
+    const bool insideWindows = false;
+#endif
+
+    // If we are outside of Windows, do not attempt to handoff

```

Comment:
```suggestion
    // If we are outside of Windows, do not attempt to hand off
```
I think `handoff` is only a noun, but you want a verb something.

Reply:
I don't really want to fix this because I use it as one word EVERYWHERE and it won't be as easy to Ctrl F...

Reply:
This fixes the grammar while preserving the noun:
```suggestion
    // If we are outside of Windows, do not attempt a handoff

```

Reply:
ok